horse/users.py

```diff
@@ -10,14 +12,21 @@
 
 
 def get_user_by_id(user_id):
-    return [u for u in users if u['id'] == user_id][0]
+    return [u for u in users if u.id == user_id][0]
+
+
+def jsonify_user(user):
```

Comment:
This method is API specific and I'd keep it away from domain models. I hope that soon we'll manage to implement robust serialization, eg. `marshmallow`